linux/gtk/src/main/java/com/github/weisj/darkmode/platform/linux/gtk/GtkSettings.kt

```diff
@@ -31,18 +31,18 @@ import com.github.weisj.darkmode.platform.settings.*
 import com.google.auto.service.AutoService
 
 @AutoService(SettingsContainerProvider::class)
-class GnomeSettingsProvider : SingletonSettingsContainerProvider({ GnomeSettings }, enabled = LibraryUtil.isGnome)
+class GtkSettingsProvider : SingletonSettingsContainerProvider({ GtkSettings }, enabled = LibraryUtil.isGtk)
 
 data class GtkTheme(val name: String) : Comparable<GtkTheme> {
     override fun compareTo(other: GtkTheme): Int = name.compareTo(other.name)
 }
 
-object GnomeSettings : DefaultSettingsContainer(identifier = "gnome_settings") {
+object GtkSettings : DefaultSettingsContainer(identifier = "gtk_settings") {
```

Comment:
This will invalidate existing stored settings. Either keep the identifier the old one with a comment explaining why or advance the settings version in https://github.com/weisJ/auto-dark-mode/blob/master/plugin/src/main/java/com/github/weisj/darkmode/AutoDarkModeOptions.kt

Reply:
ah i was not aware of this side effect. I think keeping the name with a comment is the least disruptive solution.